cosign-server/src/index.ts

```diff
@@ -0,0 +1,157 @@
+import { ethers } from "ethers";
```

Comment:
we can use https://www.npmjs.com/package/@ethersproject/solidity to reduce bundle size